CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add a view to cancel reminder mails, link added in the footer of mails
```

Comment:
commit message
✨(frontend) replace the participants stores by backend calls : replace wich by which